content/library/api/connections/connections-snowflake.md

```diff
@@ -11,6 +11,42 @@ This page only contains the `st.connections.SnowflakeConnection` class. For a de
 
 <Autofunction function="streamlit.connections.SnowflakeConnection" />
 
+## Configuration
+
+<!---
+Internal note: This section is deep-linked from the library in 1.28.1, don't break or change this anchor!
+-->
+
+`st.connection("snowflake")` can be configured using [Streamlit secrets](/library/advanced-features/secrets-management) or keyword args just like any other connection. It can also use existing Snowflake connection configuration when available.
```

Comment:
> It can also use existing Snowflake connection configuration when available.

This feels a tiny bit cryptic, but maybe I'm overthinking it since anyone using SnowflakeConnection should know what Snowflake and presumably SiS is.

I know it's explained further on, but to avoid a momentary pause of confusion, would it make sense to be a bit more concrete here?
> It can also use an existing connection configuration when running within Snowflake.

Reply:
Want to make sure we distinguish these two different scenarios (your last quote there seems to confuse them):
- When running in Snowflake (SIS), no configuration is needed. the credentials are provided by context.
- When running outside Snowflake, the configuration like connections.toml can be used

If it's not clear enough, could we either link to the anchor below or link [here again](https://docs.snowflake.com/en/developer-guide/python-connector/python-connector-example#connecting-using-the-connections-toml-file)?